hmt_escrow/eth_bridge.py

```diff
@@ -1,13 +1,13 @@
 import logging
 import os
 import time
+from typing import Any, Dict, List, Optional, Tuple
 
+from hmt_escrow.kvstore_abi import abi as kvstore_abi
```

Comment:
This reordering of variables makes this review hard to understand.

Reply:
this might have been isort

Reply:
hmm, that's actually good. let's keep it that way.